R/labeledscatterchart.R

```diff
@@ -173,6 +173,13 @@ LabeledScatter <- function(x = NULL,
             logo.urls <- rep(logo.urls, num.tables)
     }
 
+    tmp.stat <- attr(x, "statistic")
+    if ((is.array(x) || is.numeric(x)) && isTRUE(grepl("%", tmp.stat)))
+    {
+        x <- x/100
+        attr(x, "statistic") <- NULL
```

Comment:
Is the data always to be shown as a proportion?

Reply:
For all the plotly charts, percentage data is always expected to be entered as a proportion. We sent `xxx.format` (via PrepareNumbers) to indicate that the axis tick/hovertext/data labels should be shown as a percentage.